bedrock/firefox/templates/firefox/firstrun/firstrun_quantum.html

```diff
@@ -0,0 +1,78 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/onboarding" %}
+
+{% extends "firefox/firstrun/base.html" %}
+
+{% block body_id %}firstrun-quantum-onboarding{% endblock %}
+
+{% block page_title %}
+  {{ _('Welcome to Firefox') }}
```

Comment:
We need to copy all these strings in `firefox/new/quantum.lang` (not in Bedrock, on our side)